samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/CalculatedQuestionBean.java

```diff
@@ -134,6 +136,53 @@ public void removeVariable(String name) {
         variables.remove(name);
     }
 
+    // GLOBAL VARIABLES
+
+    /**
+     * getGlobalActiveVariables() returns a map of all variables that return getActive() of true
+     * @return a map of all variables that return getActive() of true.  If no global variables are active,
+     * the map will have zero elements.
+     */
+    public Map<String, CalculatedQuestionGlobalVariableBean> getGlobalActiveVariables() {
+        Map<String, CalculatedQuestionGlobalVariableBean> results = new HashMap<String, CalculatedQuestionGlobalVariableBean>();
+        for (CalculatedQuestionGlobalVariableBean globalvariable : globalvariables.values()) {
```

Comment:
this.globalVariables

Reply:
CalculatedQuestionGlobalVariableBean globalVariable

Reply:
Changed!